integration/command.go

```diff
@@ -93,72 +93,112 @@ var cleanupInspektorGadget *command = &command{
 	cleanup: true,
 }
 
-// run runs the command on the given as parameter test.
-func (c *command) run(t *testing.T) {
-	if c.startAndStop {
-		if !c.started {
-			c.start(t)
+// createExecCmd creates an exec.Cmd for the command c.cmd and stores it in
+// command.command. The exec.Cmd is configured to store the stdout and stderr in
+// command.stdout and command.stderr so that we can use them on
+// command.verifyOutput().
+func (c *command) createExecCmd() *exec.Cmd {
+	cmd := exec.Command("/bin/sh", "-c", c.cmd)
 
-			c.started = true
-		} else {
-			c.stop(t)
-		}
+	cmd.Stdout = &c.stdout
+	cmd.Stderr = &c.stderr
 
-		return
+	c.command = cmd
+
+	return c.command
+}
```

Comment:
IMO it should not return the command, it's strange that it stores and returns the command too.